Running-Mastodon/Production-guide.md

```diff
@@ -111,7 +111,14 @@ server {
 
 ## Running in production without Docker
 
-It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -u mastodon`.
+It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -Hsu mastodon`.
```

Comment:
I prefer `sudo su` but decided against it due to previous feedback

just discovered `sudo -iu` thanks to you and damn that's exactly what I wanted! thank you